src/module/token/tokenId/tokenId1155.sol

```diff
@@ -0,0 +1,89 @@
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.20;
+
+import {Module} from "../../../Module.sol";
+
+import {Role} from "../../../Role.sol";
+
+import {UpdateTokenIdCallbackERC1155} from "../../../callback/UpdateTokenIdERC1155.sol";
+import {IInstallationCallback} from "../../../interface/IInstallationCallback.sol";
+
+library TokenIdStorage {
+
+    /// @custom:storage-location erc7201:token.minting.tokenId
+    bytes32 public constant TOKEN_ID_STORAGE_POSITION =
+        keccak256(abi.encode(uint256(keccak256("token.tokenId.erc1155")) - 1)) & ~bytes32(uint256(0xff));
+
+    struct Data {
+        uint256 nextTokenId;
+    }
+
+    function data() internal pure returns (Data storage data_) {
+        bytes32 position = TOKEN_ID_STORAGE_POSITION;
+        assembly {
+            data_.slot := position
+        }
+    }
+
+}
+
+contract TokenIdERC1155 is Module, UpdateTokenIdCallbackERC1155 {
```

Comment:
would call this `SequentialTokenIdERC1155`

Reply:
updated